index.html

```diff
@@ -0,0 +1,108 @@
+		<h2>Serviços</h2>
+		<h5>Duis mollis placerat quam, eget laoreet tellus tempor eu. Quisque dapibus in purus in dignissim.</h5>
+
+		<img src="images/Service1.png" alt="">
+		<h3>Websites</h3>
+		<h5>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Reiciendis aliquid atque, nulla? Quos cum ex quis
+			soluta, a laboriosam. Dicta expedita corporis animi vero voluptate voluptatibus possimus, veniam magni quis!</h5>
+		<img src="images/Service2.png" alt="">
+		<h3>Templates</h3>
+		<h5>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Reiciendis aliquid atque, nulla? Quos cum ex quis
+			soluta, a laboriosam. Dicta expedita corporis animi vero voluptate voluptatibus possimus, veniam magni quis!</h5>
+		<img src="images/Service3.png" alt="">
+		<h3>Temas WordPress</h3>
+		<h5>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Reiciendis aliquid atque, nulla? Quos cum ex quis
+			soluta, a laboriosam. Dicta expedita corporis animi vero voluptate voluptatibus possimus, veniam magni quis!</h5>
+	</div>
+
+	<!-- Meus Trabalhos -->
+	<div class="">
+		<h2>Meus trabalhos</h2>
+		<h5>Duis mollis placerat quam, eget laoreet tellus tempor eu. Quisque dapibus in purus in dignissim.</h5>
+		<button type="button" name="btnTudo">Tudo</button>
+		<button type="button" name="btnWebSite">Website</button>
+		<button type="button" name="btnTemplate">Template</button>
+		<button type="button" name="btnTema">Tema</button>
+		<br>
+		<img src="images/Portfolio01.png" alt="">
+		<h5>Thumbnail label</h5>
+		<p>Thumbnail caption</p>
+		<img src="images/Portfolio02.png" alt="">
+		<h5>Thumbnail label</h5>
+		<p>Thumbnail caption</p>
+		<img src="images/Portfolio03.png" alt="">
+		<h5>Thumbnail label</h5>
+		<p>Thumbnail caption</p>
+	</div>
+
+	<!-- Quem Sou Eu -->
+	<div class="">
+		<h2>Quem sou eu</h2>
+		<h5>Duis mollis placerat quam, eget laoreet tellus tempor eu. Quisque dapibus in purus in dignissim.</h5>
+		<img src="images/photo.png" alt="">
+		<br>
+		<caption>Joana D'arc</caption>
+		<br>
+		<p>ícones </p>
+		<h2>Sobre Mim</h2>
+		<h5>Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diamnonummy nibh euismod tincidunt ut laoreet
+			dolore magna aliquam erat volutpat. Ut wisi enim ad minim veniam, quis nostrud exerci tation ullamcorper suscipit
+			lobortis nisl ut aliquip ex ea commodo consequat. Duis autem vel eum iriure dolor in hendrerit in vulputate velit
+			esse molestie consequat, vel illum dolore eu feugiat nulla facilisis at vero eros et accumsan et iusto odio
+			dignissim qui blandit praesent luptatum zzril delenit augue duis dolore te feugait nulla facilisi.</h5>
+		<h2>Habilidades</h2>
+		<li>Java Script</li>
+		<li>Html & CSS</li>
+		<li>jQuery</li>
+		<li>Wordpress</li>
+	</div>
+
+	<!-- Contato -->
+	<div class="">
+		<h2>Contato</h2>
+		<h5>Duis mollis placerat quam, eget laoreet tellus tempor eu. Quisque dapibus in purus in dignissim.</h5>
+		<h4>meuemail@mail.com</h4>
+		<h4>+55 21 987654-321</h4>
```

Comment:
Mesma coisa do email também vale para telefones.
Referência: https://www.w3schools.com/tags/tag_a.asp

```suggestion
		<a href="tel:+5521987654-321">+55 21 987654-321</a>
```